Fixing Pay Invoice and Send Invoice articles

Christinadobrzyn opened the pull request:
### Details

These two articles look bad:

https://help.expensify.com/articles/new-expensify/expenses-&-payments/pay-an-invoice
https://help.expensify.com/articles/new-expensify/expenses-&-payments/Send-an-invoice

I'm hoping this PR will fix them. Per this [Slack convo](https://expensify.slack.com/archives/C02QSAC6BJ8/p1724609654169599) I added "{% include end-selector.html %}" below the mobile and desktop sections. 


### Fixed Issues
Hopefully I fixed the formatting issue so that the article looks correct. See screenshots below of what it currently looks like. 

I think there will probably be other things that need to be fixed - it looks like the links are working but if we can at least fix the formatting then I can figure out what else needs to be fixed. 

### PR Author Checklist
<!--
This is a checklist for PR authors. Please make sure to complete all tasks and check them off once you do, or else your PR will not be merged!
-->

- [X] I linked the correct issue in the `### Fixed Issues` section above
- [X] I wrote clear testing steps that cover the changes made in this PR
    - [X] I added steps for local testing in the `Tests` section
    - [X] I added steps for the expected offline behavior in the `Offline steps` section
    - [X] I added steps for Staging and/or Production testing in the `QA steps` section
    - [X] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [X] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [X] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [X] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [X] I ran the tests on **all platforms** & verified they passed on:
    - [X] Android: Native
    - [X] Android: mWeb Chrome
    - [X] iOS: Native
    - [X] iOS: mWeb Safari
    - [X] MacOS: Chrome / Safari
    - [X] MacOS: Desktop
- [X] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [X] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [X] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [X] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [X] I verified that comments were added to code that is not self explanatory
    - [X] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [X] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [X] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [X] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [X] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [X] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [X] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [X] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [X] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [X] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [X] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [X] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [X] I verified that if a function's arguments changed that all usages have also been updated correctly
- [X] If any new file was added I verified that:
    - [X] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [X] If a new CSS style is added I verified that:
    - [X] A similar style doesn't already exist
    - [X] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [X] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [X] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [X] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [X] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [X] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [X] I verified that all the inputs inside a form are aligned with each other.
    - [X] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [X] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [X] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<img width="1154" alt="image" src="https://github.com/user-attachments/assets/c1c197ca-fe4d-49b8-80b6-5c575791f64d">

<img width="1123" alt="image" src="https://github.com/user-attachments/assets/200e1d16-41c4-4c0f-a997-ffeaa5ff60d2">
</details>

Christinadobrzyn committed:
Update Send-an-invoice.md

trying to fix an issue that's breaking the article format

Christinadobrzyn committed:
Update pay-an-invoice.md

Trying to fix a formatting issue that's breaking the article. Added "{% include end-selector.html %}" below the Mobile and Desktop settings

OSBotify commented:
A preview of your ExpensifyHelp changes have been deployed to https://8a741651.helpdot.pages.dev ⚡️
<!-- Created by actions-cool/maintain-one-comment -->

blimpich commented:
@Christinadobrzyn can you confirm using the [preview of your changes](https://github.com/Expensify/App/pull/48012#issuecomment-2310529645) that this looks the way you expect it to?

Christinadobrzyn commented:
@blimpich thanks for referencing the preview, no it doesn't look right. Everything is grouped by the mobile or desktop. should I create a new PR to try and fix this? The article should look like this: https://8a741651.helpdot.pages.dev/articles/new-expensify/expenses-&-payments/Create-an-expense

blimpich commented:
Ah I see, lemme take a look at it for a minute and see if I can figure out why it looks so wonky

Christinadobrzyn committed:
Update Approve-and-pay-expenses.md

Update a formatting issue at the top of the article- checked preview and it looks right.

blimpich commented on docs/articles/new-expensify/expenses-&-payments/Send-an-invoice.md:
```diff
@@ -26,7 +26,7 @@ Workspace admins can enable invoicing on a workspace to send invoices and receiv
 4. Tap **More features**. 
 5. Under the Earn section, enable the Invoice toggle.
 
-{% include end-option.html %}
```
Lets not delete this.

blimpich commented on docs/articles/new-expensify/expenses-&-payments/pay-an-invoice.md:
```diff
@@ -6,7 +6,8 @@ description: process to pay an invoice
 
 {% include info.html %}
 Anyone who receives an Expensify invoice can pay it using Expensify—even if they don’t have an Expensify account.
-{% include end-info.html %}
```
I don't think we should delete this.

blimpich commented on docs/articles/new-expensify/expenses-&-payments/pay-an-invoice.md:
```diff
@@ -19,16 +20,19 @@ To pay an invoice,
 2. Click **Pay**.
 3. Choose **Pay as an individual** or **Pay as a business**.
 4. Click **Add Bank Account** or **Add debit or credit card** to issue payment. 
-{% include end-option.html %}
```
Same here, don't think this should be deleted.

blimpich commented on docs/articles/new-expensify/expenses-&-payments/pay-an-invoice.md:
```diff
@@ -19,16 +20,19 @@ To pay an invoice,
 2. Tap **Pay**.
 3. Choose **Pay as an individual** or **Pay as a business**.
 4. Tap **Add Bank Account** or **Add debit or credit card** to issue payment.
-{% include end-option.html %}
```
No delete here too.

blimpich reviewed (changes requested):
Okay I think I understand. @Christinadobrzyn I think what @rushatgabhane meant by [this comment](https://expensify.slack.com/archives/C02QSAC6BJ8/p1724676873521279?thread_ts=1724609654.169599&cid=C02QSAC6BJ8) is that we are missing a `{% include end-selector.html %}` from the document. This PR does that but it also removes multiple `{% include end-option.html %}`.

They are basically acting like HTML tags, so for every time `{% include selector.html values="desktop, mobile" %}` shows up that means that there needs to be a corresponding `{% include end-selector.html %}` below it to act as a kind of closing tag. Sort of like Russian nesting dolls.

@Christinadobrzyn can you go through these documents again and make sure that your changes are simply adding the missing `{% include end-selector.html %}`? Your changes to the `end-option` tags I think are breaking more things.

blimpich commented:
And if its easier for you just create a new PR and add me as the reviewer 👍

Christinadobrzyn commented:
Okay, I created a new PR - https://github.com/Expensify/App/pull/48022 If you can take a peek. TY!
